chat-plugins/github.js

```diff
@@ -0,0 +1,94 @@
+/*
+	Github plugin by Spandamn
+	This is based on https://github.com/Ecuacion/Pokemon-Showdown-Node-Bot/blob/master/features/github/index.js
+	Yes, some of the code is copied so credits to xfix, Ecuacion, jd, and anyone who's name I forgot to put here
+*/
+'use strict';
+
+let config = {};
+
+if (!Config.github) {
+	return;
+} else if (Config.github === {}) {
+	config.port = 3420;
```

Comment:
Blergh, just putting it out there, but for me since I host it on C9 I had to use a port like 8080, 8081, 8082 (whichever one your config port isn't set to..... Perhaps you could comment that for it? Just so people can use this with ease haha